src/lib.rs

```diff
@@ -6,4 +6,5 @@ pub mod render;
 pub mod scan;
 
 #[cfg(test)]
+#[cfg(feature = "pre-render")]
```

Comment:
The escape-mode test and the tests not to do with rendering should be on unconditionally.